### ISSUE TITLE
TypeError: Object function dbug... has no method 'split'

### ISSUE THREAD

**Issue by rvadhavk** [seanmonstar/dbug#3]
index.js:245
`if (decimalVersion(dbug) > decimalVersion(global.__dbug__)) {`
should be 
`if (decimalVersion(dbug.version) > decimalVersion(global.__dbug__.version)) {`

**Comment by shane-tomlinson** [comment]
@seanmonstar - I'm seeing this too when trying to add `mozlog` to mozilla/fxa-content-server. :( I'm running on OSX.

**Comment by shane-tomlinson** [comment]
@seanmonstar - if I add the following snippet to `decimalVersion`, I see strangeness in the output:

``` js
console.log("typeof verStr: " + typeof verStr);
```

``` bash
typeof verStr: function
```

**Comment by seanmonstar** [comment]
I totally missed this issue, sorry.

@shane-tomlinson wtf. can you console.trace()?

**Comment by seanmonstar** [comment]
Yikes, how embarrassing. Someone submitted a fix for this, I merged it, and then forgot all about it. I've just published [v0.4.2](https://github.com/seanmonstar/dbug/releases/tag/v0.4.2) that includes the fix.

**Comment by shane-tomlinson** [comment]
Thanks @seanmonstar!
